lib/project_types/script/layers/application/project_dependencies.rb

```diff
@@ -14,7 +14,7 @@ def self.install(ctx:, task_runner:)
                 end
               end
               true
-            rescue Infrastructure::Errors::DependencyInstallError => e
+            rescue Infrastructure::Errors::DependencyInstallationError => e
```

Comment:
I know line 19 isn't your change, but I don't know if we should do it that way.  We should be using UI::ErrorHandler, instead of extracting the message from the Error class here.  If you agree with it, then can you please put the code in a better place than when you found it, and make the change in this PR?

Reply:
I think line 19 is still the right approach. It was part of our designs because we want to make sure all of the output is visible to the user while keeping the error output aligned with where the failure occurred (in our case, within the dependency installation frame).

This is an example of what line 19 prints when a dependency fails to install:

![image](https://user-images.githubusercontent.com/28009669/130095837-df4fb92e-8d3f-4cc9-bdfe-175688711be5.png)